test/e2e/scheduling/predicates.go

```diff
@@ -936,7 +940,11 @@ func WaitForSchedulerAfterAction(f *framework.Framework, action Action, ns, podN
 	}
 	success, err := observeEventAfterAction(f.ClientSet, f.Namespace.Name, predicate, action)
 	framework.ExpectNoError(err)
-	framework.ExpectEqual(success, true)
+	if success {
```

Comment:
This still doesn't seem correct.

I think what is causing confusion is line 941. The variable should be `observed` or `found` instead of `success`.

The message should be based on `expectSuccess`.

Reply:
@alculquicondor  what about now?